ecoscope/analysis/__init__.py

```diff
@@ -1,9 +1,3 @@
-from ecoscope.analysis import UD, astronomy, seasons
-from ecoscope.analysis.classifier import apply_classification
-from ecoscope.analysis.ecograph import Ecograph, get_feature_gdf
-from ecoscope.analysis.percentile import get_percentile_area
-from ecoscope.analysis.speed import SpeedDataFrame
-
 __all__ = [
     "Ecograph",
     "SpeedDataFrame",
```

Comment:
If we're removing these imports do we also need to change the `__all__` assignment here?

Reply:
Yep, this is something I missed. I'll fix this and maybe add a test to cover wildcard imports.